optimizely/decision_service.py

```diff
@@ -34,20 +35,28 @@ def __init__(self, config, user_profile_service):
     self.config = config
     self.logger = config.logger
 
-  @staticmethod
-  def _get_bucketing_id(user_id, attributes):
+  def _get_bucketing_id(self, user_id, attributes):
     """ Helper method to determine bucketing ID for the user.
 
     Args:
       user_id: ID for user.
       attributes: Dict representing user attributes. May consist of bucketing ID to be used.
 
     Returns:
-      String representing bucketing ID for the user. Fallback to user's ID if not provided.
+      String representing bucketing ID for the user. Fallback to user's ID if not provided or is not a string.
```

Comment:
Get the bucketingId of a user if a bucketingId exists in attributes and type is string.

Reply:
I have copied this comment from Java SDK, that Mike suggested in it's PR.